core/modules/contact/contact.admin.inc

```diff
@@ -22,6 +22,17 @@ function contact_settings_form($form, &$form_state) {
     '#collapsed' => FALSE,
     '#title' => t("Fields"),
   );
+  $form['fields']['phone'] = array(
+    '#type' => 'select',
+    '#title' => t('Phone number'),
+    '#description' => t('Allow users to enter their phone number.'),
+    '#options' => array(
+      'optional' => t('Optional'),
+      'required' => t('Required'),
+    ),
+    '#empty_value' => '',
+    '#default_value' => $config->get('phone'),
+  );
   $form['fields']['copy'] = array(
```

Comment:
Perhaps this is not within the scope of this issue but I don't get why this fieldset **was** called fields before this PR. It actually makes sense now that we have the phone field in there, but now I don't think the "copy" checkbox fits.

Reply:
The 'copy' checkbox *is* actually a field that appears on the Contact form though:

![image](https://user-images.githubusercontent.com/2385329/98186861-cf97fd80-1f63-11eb-9391-db871fe29bce.png)

Reply:
Ah, truth. Ignore me.